pkg/controller/volume/attachdetach/attach_detach_controller.go

```diff
@@ -310,6 +311,8 @@ func (adc *attachDetachController) processPodVolumes(
 	nodeName := types.NodeName(pod.Spec.NodeName)
 
 	if !adc.desiredStateOfWorld.NodeExists(nodeName) {
+		// TODO: Is this a race between the node watcher & the pod watcher?
+
```

Comment:
If this happens, it still ok since reconciler will add the pod to desired state in next round after node is added to the state.